### ISSUE TITLE
Visual Cues for Data Analysis Features Amplify Challenges for Blind Spreadsheet Users

### ISSUE THREAD

**Issue by Masa630** [Wotipati/Blind_Accessibility_Papers#611]
### Links
- Paper: https://dl.acm.org/doi/pdf/10.1145/3613904.3642753

### Abstract
Spreadsheets are widely used for storing, manipulating, analyzing, and visualizing data. Featuressuch as conditional formatting, formulas, sorting, and fltering play an important role when understanding and analyzing data in spreadsheets. They employ visual cues, but we have little understanding of the experiences of blind screen reader (SR) users with such features. We conducted a study with 12 blind SR users to gain insights into their challenges, workarounds, and strategies in understanding and extracting information from a spreadsheet consisting of multiple tables that incorporated data analysis features. We identifed fve factors that impact blind SR users’ experiences: cognitive overload, time-information trade-of, lack of awareness and expertise, inadequate system feedback, and delayed and absent SR responses. Drawn from these fndings, we discuss design suggestions and future research agenda to improve SR users’ spreadsheet experiences.

<img width="891" alt="スクリーンショット 2024-06-13 18 05 07" src="https://github.com/SeitaKayukawa/Blind_Accessibility_Papers/assets/127022571/9f5d9d29-52cc-4af0-933c-47e0a628b203">

視覚障害者がスプレッドシートを利用する際に直面する課題について調査した論文
Contribution:
1.Identifcation of the strategies blind spreadsheet users employ
2.Deeper understanding of the difculties
3.Discussion of the future research avenues and design considerations

12名の視覚障害者を対象に、スプレッドシートを探索した後に5個の質問に答えるユーザ実験を行った。
困難な点としては
・探しているコンテンツの特定
・テーブルの構造の把握
・視覚的な手がかりの特定と理解
・セルの参照を用いた数式の作成
